studio/schemas/objects/link.ts

```diff
@@ -190,10 +192,12 @@ export const link = defineField({
       title: "linkTitle",
       type: "linkType",
     },
-    prepare(selection) {
-      const { title, type } = selection;
+    prepare({ title, type }) {
+      const translatedTitle = isInternationalizedString(title)
+        ? firstTranslation(title)
+        : null;
       return {
-        title: title[0].value,
+        title: translatedTitle ?? "Missing title",
```

Comment:
``` suggestion
        title: title[0].value || null,
```

Reply:
See comment above, but returning `null` is probably better than `"Missing title"` :+1: